pkg/csi/driver/controller.go

```diff
@@ -169,6 +169,28 @@ func (d *ControllerDriver) CreateVolume(ctx context.Context, req *csi.CreateVolu
 		return nil, status.Errorf(codes.InvalidArgument, "failed to parse storageclass parameters %v", err)
 	}
 
+	var sourceOcid string
+	sourceVolume := req.GetVolumeContentSource()
+	log.Info("Volume Content Source: ", sourceVolume)
+
+	if sourceVolume != nil {
+		sourceOcid = sourceVolume.GetVolume().VolumeId
```

Comment:
`req.GetVolumeContentSource().GetVolume()` can return `nil`. Please add a check for `sourceVolume.GetVolume()` being not `nil`.